Assets/Scripts/ForceQuit.cs

```diff
@@ -0,0 +1,14 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ForceQuit : MonoBehaviour
+{
+    void Update()
+    {
+        if(Input.GetKey(KeyCode.Escape) && Input.GetKey(KeyCode.X))
+        {
+            Application.Quit();
```

Comment:
Thanks for adding this in. Helps our the Arcade team with being able to showcase different games on their machine in an easier way.